device-handler/FibaroMotionSensor/FibaroMotionSensor.groovy

```diff
@@ -100,14 +100,14 @@ metadata {
             ]
         }
         valueTile("illuminance", "device.illuminance", inactiveLabel: false) {
-            state "luminosity", label:'${currentValue} ${unit}', unit:"lux"
+            state "luminosity", label:'${currentValue} lux', unit:"lux"
         }
         valueTile("battery", "device.battery", inactiveLabel: false, decoration: "flat") {
             state "battery", label:'${currentValue}% battery', unit:""
         }
         standardTile("acceleration", "device.acceleration") {
-            state("vibration", label:'${currentValue}', icon:"st.motion.acceleration.active", backgroundColor:"#53a7c0")
-            state("still", label:'${currentValue}', icon:"st.motion.acceleration.inactive", backgroundColor:"#ffffff")
+            state("active", label:'Vibration', icon:"st.motion.acceleration.active", backgroundColor:"#53a7c0")
+            state("inactive", label:'Still', icon:"st.motion.acceleration.inactive", backgroundColor:"#ffffff")
         }
```

Comment:
Initially, my unit reported 'inactive'.   After your comment, I knocked on the sensor to trigger the event and it did indeed show numbers (acceleration), and then finally 'still'.    
I'd suggest keeping your code, but adding the one state for 'inactive' to cover that one end case.